### DISCUSSION TITLE
1.15+离线账户使用正版皮肤时无法保存世界

### DISCUSSION THREAD

**Discussion by pholt0** [Meloong-Git/PCL#4840] [提问] [answered]
1.15+离线账户使用正版皮肤时无法保存世界

**Comment by MoYuan-CN** [comment]
提供一下日志？

**Reply by pholt0** [reply]
会卡在保存世界中，但是如果ID和正版玩家名一样的话就不会有问题

**Reply by MoYuan-CN** [reply]
提供你的游戏日志[^1]，不然我们无法分析任何问题
<img src="https://github.com/user-attachments/assets/3a89b108-3f74-4052-850b-437ec11c9a76" width="240">

[^1]: PCL 启动器中：版本设置 -> 快捷方式：版本文件夹 -> `logs` 文件夹 -> lastet.log 或者包含问题发生的日志压缩包

**Comment by LTCatt** [comment, marked as answer]
#3746 ，更新一下 PCL，或者发日志
